blint/lib/utils.py

```diff
@@ -207,6 +216,37 @@ def is_ignored_file(file_name):
     return any(file_name.endswith(ie) for ie in ignore_files)
 
 
+def blintdb_setup(args):
+    """
+    This function downloads blint-db package from 'ghcr.io/appthreat/blintdb-vcpkg' using oras client
+    and puts it into $BLINTDB_LOC path.
+    If there is not path in $BLINTDB_LOC, it will add it to $HOME/blindb.
+    $USE_BLINTDB is required to be set "true" or "1", in order to use blintdb
+    """
+    if not os.getenv("USE_BLINTDB") or not args.use_blintdb :
+        LOG.debug(f"skipping blintdb setup {os.getenv('USE_BLINTDB')}")
```

Comment:
@aryan-rajoria Please capitalize the first letter of log messages for consistency.